// Copyright 2025 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
syntax = "proto2";

package test_namespace;

import "py/arolla/codegen/testing/example_codegen_function.proto";

message FooExtension {
  extend test_namespace.FooInput.BarInput {
    optional FooExtension extension_foo = 100000;
  }
  optional FooInput foo = 1;
}

message OutputExtension {
  extend test_namespace.Output {
    optional OutputExtension extension_output = 100000;
  }
  optional float extra_result = 1;
}
